zebrad/src/commands/start.rs

```diff
@@ -67,3 +131,123 @@ impl config::Override<ZebradConfig> for StartCmd {
         Ok(config)
     }
 }
+
+struct ZebraNode<ZN, ZS>
```

Comment:
Maybe this could be called `Core` rather than `Node` to avoid ambiguity with `Network` (`ZN`)?